doc/ref_model/chapters/chapter06.rst

```diff
@@ -60,6 +107,58 @@ The Or-Vi and Vi-Vnfm are both specifying interfaces provided by the VIM and the
++--------------+------+----+------+------+------+----------------------------------------------------------------------+
+| Privileges   |      |    |      |      |      | Created and managed by Cloud Service Provider(CSP) administrators    |
++--------------+------+----+------+------+------+----------------------------------------------------------------------+
+| Role         | +    | +  |      |      | +    | Create/delete by authorized administrators where roles are assigned  |
+|              |      |    |      |      |      | privileges and mapped to users in scope                              |
++--------------+------+----+------+------+------+----------------------------------------------------------------------+
+| Security     | +    | +  |      |      | +    | Create and delete only by VDC administrators                         |
+| Groups       |      |    |      |      |      |                                                                      |
++--------------+------+----+------+------+------+----------------------------------------------------------------------+
+| Stack        | +    | +  |      |      | +    | Create/delete by VDC users with appropriate role                     |
++--------------+------+----+------+------+------+----------------------------------------------------------------------+
+| Virtual      | +    | +  | +    | +    | +    | Create/delete by VDC users with appropriate role                     |
+| Storage      |      |    |      |      |      |                                                                      |
++--------------+------+----+------+------+------+----------------------------------------------------------------------+
+| User         | +    | +  |      | +    | +    | Create/delete only by VDC administrators                             |
++--------------+------+----+------+------+------+----------------------------------------------------------------------+
+| Tenant       | +    | +  |      | +    | +    | Create/delete only by Cloud Zone administrators                      |
++--------------+------+----+------+------+------+----------------------------------------------------------------------+
+| Virtual      | +    | +  |      | +    | +    | Create/delete by VDC users with appropriate role. Additional         |
+| compute      |      |    |      |      |      | operations would include suspend/unsuspend                           |
++--------------+------+----+------+------+------+----------------------------------------------------------------------+
+| Virtual      | +    | +  | +    | +    | +    | Create/delete by VDC users with appropriate role                     | 
+| network      |      |    |      |      |      |                                                                      |
++--------------+------+----+------+------+------+----------------------------------------------------------------------+
+=======
```

Comment:
remove

Reply:
Fixed in [f39b95f](https://github.com/cntt-n/CNTT/pull/2931/commits/f39b95f9a9cfbd4d02bbb624817269c76e05707e)